src/templates/InstantSearch iOS/App/ViewController.swift

```diff
@@ -26,7 +26,7 @@ class ViewController: HitsTableViewController {
 
         cell.textLabel?.highlightedTextColor = .blue
         cell.textLabel?.highlightedBackgroundColor = .yellow
-        cell.textLabel?.highlightedText = SearchResults.highlightResult(hit: hit, path: "name")?.value
+        cell.textLabel?.highlightedText = SearchResults.highlightResult(hit: hit, path: "{{attributesToDisplay.[0]}}")?.value
```

Comment:
I'm not sure how to handle multiple attributes UI-wise in Swift.

Reply:
Let's make that an issue and ask some help from the mobile team #178